modules/tests/kubecf-test.sh

```diff
@@ -96,10 +96,23 @@ kubectl logs -f "${pod_name}" --namespace "${KUBECF_NAMESPACE}" --container "$co
 
 # Wait for the container to terminate and then exit the script with the container's exit code.
 jsonpath='{.status.containerStatuses[?(@.name == "'"$container_name"'")].state.terminated.exitCode}'
-while true; do
-  exit_code="$(kubectl get "${pod_name}" --namespace "${KUBECF_NAMESPACE}" --output "jsonpath=${jsonpath}")"
-  if [[ -n "${exit_code}" ]]; then
-      exit "${exit_code}"
-  fi
-  sleep 1
+exit_code="$(kubectl get "${pod_name}" --namespace "${KUBECF_NAMESPACE}" --output "jsonpath=${jsonpath}")"
+while [[ -z "${exit_code}" ]]; do
+    exit_code="$(kubectl get "${pod_name}" --namespace "${KUBECF_NAMESPACE}" --output "jsonpath=${jsonpath}")"
+    sleep 1
 done
+
+# save results of tests to file
+mkdir -p artifacts
+kubectl logs "${pod_name}" --namespace "${KUBECF_NAMESPACE}" --container "$container_name" > \
+        artifacts/"$(date +'%Y-%m-%d-%H:%M')"_"$( echo "${pod_name}" | cut -d '/' -f 2)".log
```

Comment:
This line is a bit long; maybe use a temporary variable?
```suggestion
log="artifacts/$(date +'%Y-%m-%d-%H:%M')_${pod_name#*/}.log"
kubectl logs "${pod_name}" --namespace "${KUBECF_NAMESPACE}" --container "$container_name" > "${log}"
        
```

Reply:
way cleaner, and with var substitution, thanks.